cmd/leanix-k8s-connector/main.go

```diff
@@ -78,10 +72,7 @@ func KubernetesScan(debugLogBuffer *bytes.Buffer) (response *leanix.SelfStartRes
 	accessToken, err := leanix.Authenticate(viper.GetString(integrationAPIFqdnFlag), viper.GetString(integrationAPITokenFlag))
 	if err != nil {
 		log.Error(err)
-		log.Info("Failed to authenticate. Terminating..")
-		return nil, err
```

Comment:
Could you pull in the latest develop changes

Reply:
I will check, I thought I did, but maybe this one got overwritten.